docs: additional dependencies for running Docling in Linux-based Docker environments (#10861)
langflow-ai/langflow: docs/docs/Support/troubleshooting.mdx

Token length errors can happen if your chunking strategy doesn't align with your embedding model's tokenization limits.
For more information, see [Tokenization errors due to chunk size](/split-text#chunk-size).

## Document processing errors in Docker containers

> If you're running Langflow in a Docker container and encounter errors when using components that handle files or images, such as the [**Read File** component](/read-file) or [**Docling** components](/bundles-docling), you may need to install additional system dependencies.

Reviewer: ```suggestion
If you're running Langflow in a Docker container, you might encounter errors when using components that handle files or images, such as the [**Read File** component](/read-file) or [**Docling** components](/bundles-docling). To resolve these errors, you might need to install additional system dependencies.
```
